.azure/scripts/setup-kind.sh

```diff
@@ -198,34 +287,31 @@ elif [[ "$IP_FAMILY" = "ipv6" ]]; then
      # use ULA (i.e., Unique Local Address), which offers a similar "private" scope as link-local
     # but without the interface dependency and some of the other challenges of link-local addresses.
     # (link-local starts as fe80::) but we will use ULA fd01
-    updateDockerDaemonConfiguration "{
-        \"insecure-registries\" : [\"[${ula_fixed_ipv6}::1]:${reg_port}\", \"${registry_dns}:${reg_port}\"],
-        \"experimental\": true,
-        \"ip6tables\": true,
-        \"fixed-cidr-v6\": \"${ula_fixed_ipv6}::/80\"
-    }"
-
-    cat <<EOF | kind create cluster --image "${KIND_NODE_IMAGE}" --config=-
-    kind: Cluster
-    apiVersion: kind.x-k8s.io/v1alpha4
-    nodes:
-    - role: control-plane
-    - role: control-plane
-    - role: control-plane
-    - role: worker
-    - role: worker
-    - role: worker
-    name: $KIND_CLUSTER_NAME
-    containerdConfigPatches:
-    - |-
-      [plugins."io.containerd.grpc.v1.cri".registry.mirrors."myregistry.local:5001"]
-          endpoint = ["http://myregistry.local:5001"]
-    networking:
-        ipFamily: $IP_FAMILY
+    updateContainerRuntimeConfiguration "" "${ula_fixed_ipv6}" "${reg_port}" "${registry_dns}"
+
+    cat <<EOF | sudo systemd-run --scope -p "Delegate=yes" /usr/local/bin/kind create cluster --image "${KIND_NODE_IMAGE}" --name "${KIND_CLUSTER_NAME}" --config=-
```

Comment:
why full path to kind binary is needed? Few lines above you did step `Move the binary to a globally accessible location` ?

Reply:
yeah that's not needed let me fix this :) thanks